iosApp/Survey/Sources/Supports/Helpers/KMM/AutoMockable/Shared+AutoMockable.swift

```diff
@@ -78,3 +78,13 @@ protocol LogOutUseCaseKMM: LogOutUseCase {
         @escaping (Error?, KotlinUnit) -> KotlinUnit
     ) -> () -> KotlinUnit
 }
+
+// sourcery: AutoMockable
+protocol SurveyListUseCaseKMM: SurveyListUseCase {
+
+    func invoke(page: Int32) -> Kotlinx_coroutines_coreFlow
```

Comment:
Just curious, can we just use `Int` here? Kotlin also supports `Int` type, right? 🤔

Reply:
`Int` in KMM is converted to Swift `Int32`. 😅 The method signature generated by framework here takes `Int32`.